tests/installation/playwright-cli-install-should-work.spec.ts

```diff
@@ -95,3 +95,27 @@ test('install playwright-chromium should work', async ({ exec, installedSoftware
   await exec('npx playwright install chromium');
   await exec('node sanity.js playwright-chromium chromium');
 });
+
+test('should print error if recording video without ffmpeg', async ({ exec }) => {
```

Comment:
Do you want to use custom browser path and delete ffmpeg from that path?  otherwise I don't understand this test.

Reply:
We have isolated browsers in this test so Playwright does not find any browsers including ffmpeg. I need a browser to start with tho - for that I use Google Chrome. We have other tests relying on Google Chrome as well.

Reply:
done.